Packages/Netherlands3D/T3DPipeline/Runtime/Scripts/CityJSONExporter/CityJSONFormatter.cs

```diff
@@ -102,15 +102,22 @@ private static JSONArray GetGeographicalExtents(Dictionary<Vector3Double, int> v
             return extentArray;
         }
 
-        //register city object to be added to the JSON when requested
-        public static void AddCityObject(CityObject obj)
+        //register city object to be added to the JSON when requested. Returns whether successful or not
+        public static bool AddCityObject(CityObject obj)
         {
+            if (CityObjects.Contains(obj))
+                return false;
             CityObjects.Add(obj);
+            return true;
+
         }
 
-        public static void RemoveCityObject(CityObject obj)
+        public static bool RemoveCityObject(CityObject obj)
         {
+            if (!CityObjects.Contains(obj))
+                return false;
```

Comment:
bij een single line if statement de volgende regel denk ik een newline doen

Reply:
gefixt in nieuwe commit